cmd/limactl/main_ext.go

```diff
@@ -0,0 +1,7 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+// Import ext driver to register it in the registry on all platforms.
+import _ "github.com/lima-vm/lima/v2/pkg/driver/ext"
```

Comment:
Should this be compiled as an external driver? (w/ docs)

Reply:
Currently that is not possible, but might be an option (like the other built-in ones)

Reply:
https://github.com/lima-vm/lima/pull/2000#issuecomment-3262065114